src/docfx/build/legacy/metadata/LegacyMetadata.cs

```diff
@@ -68,6 +69,16 @@ public static JObject GenerateLegacyRawMetadata(PageModel pageModel, Docset docs
                 rawMetadata["redirect_document_id"] = redirectionMap.TryGetDocumentId(file, out var _);
             }
 
+            var culture = new CultureInfo(pageModel.Locale);
+            rawMetadata["_op_gitContributorInformation"] = new JObject
+            {
+                ["author"] = JObject.FromObject(pageModel.Author),
+                ["contributors"] = JObject.FromObject(pageModel.Contributors),
+                ["update_at"] = pageModel.UpdatedAt.ToString(culture.DateTimeFormat.ShortDatePattern, culture),
+            };
+            rawMetadata["author"] = pageModel.Author.ProfileUrl.Substring(pageModel.Author.ProfileUrl.LastIndexOf('/'));
+            rawMetadata["updated_at"] = pageModel.UpdatedAt.ToString("yyyy-MM-dd hh:mm tt", culture);
```

Comment:
>  [](start = 86, length = 1)

missing `:`?

Reply:
it should be same as before

---
In reply to: [198001445](https://github.com/dotnet/docfx/pull/2912#discussion_r198001445) [](ancestors = 198001445)

Reply:
sorry I overlooked. you are right :)

---
In reply to: [198005670](https://github.com/dotnet/docfx/pull/2912#discussion_r198005670) [](ancestors = 198005670,198001445)